messaging/views.py

```diff
@@ -6,12 +6,18 @@
 from users.models import Member
 from .models import Message, MessageThread
 from django.contrib.auth.models import User
+from notifications.models import notify
 
 import re
 
 # not a view
 # helper function to send messages
 def send_message(member, thread, message):
+	# Send the notification to everyone in the thread except the sender.
+	url = reverse('message_thread', args=[thread.id])
+	for receiver in thread.participants.all():
+		if member != receiver:
+			notify(receiver, 'message_received', 'You got a message from '+str(member.equiv_user.username)+': '+message, url)
```

Comment:
I think it might be nice to move the notification names out into notifications, as a fake-enum

Reply:
So instead we'd go like,
```
notify(receiver, notifications.MESSAGE_RECEIVED, 'What a cool notification', url)
```

Reply:
The reason I didn't do this is so that undefined notification types could be sent, and they'd display with the default icon (I think it's fa-circle). I think this is possibly less necessary now that every notification type has been implemented, but not having to import something else is nice. Thoughts?

Reply:
That's fair.

I don't think we want to send undefined notification types - they should be an existing type or None - otherwise they could be mistaken for (or could be) a mistakenly incorrect name.

Reply:
Completed. :racehorse: